.github/workflows/quality.yml

```diff
@@ -0,0 +1,107 @@
+name: Code quality with Pylint & iSort
+
+on:
+  # Trigger workflow on pull request, but only for PRs merging into zahid/LP-2361-from-dev branch
+  pull_request:
+    types: [opened, synchronize, reopened]
+    branches:
+      - master
```

Comment:
do we need to run linter on master branch when it has already run on develop branch?

Reply:
We can skip it for master branch, if you want.

Reply:
@zahid-arbisoft @emraan004 did we decide it?

Reply:
@ahmed-faraz46 going with both branches